test/integration/Mayan.t.sol

```diff
@@ -19,6 +20,10 @@ contract MayanTest is BridgeSettlerIntegrationTest {
     address forwarder = 0x337685fdaB40D39bd02028545a4FfA7D287cC3E2;
     address mayanProtocol;
 
+    function testBridgeSettler() internal override {
+        bridgeSettler = new ArbitrumBridgeSettler(bytes20(0));
+    }
+
```

Comment:
What is the point of this function? The EVM environment is reset between tests, so all that this is really testing for is the fact that `ArbitrumBridgeSettler`'s constructor doesn't revert

EDIT: I see that this function is `internal` and that you call it from `setUp()`. consider naming this function something different. it's confusing

Reply:
well shame on me for establishing that pattern :sweat_smile: a leading underscore is probably fine